gradle.properties

```diff
@@ -19,14 +18,16 @@ minecraft_version_range=[1.21.1]
 # options are: alpha, beta, release
 release_type=alpha
 
+# Enable or disable the use of dev logon for the client run
+enable_devlogin=false
```

Comment:
This shouldn't be here, we should instead have a comment saying to place this property in the user's gradle properties file.